pkg/convert/pot_test.go

```diff
@@ -224,7 +243,8 @@ func Test_ToPot(t *testing.T) {
 			if !assert.NoError(t, err) {
 				return
 			}
-
+			fmt.Printf("exp: %v\n", string(tt.expected))
+			fmt.Printf("res: %v\n", string(result))
```

Comment:
Remove print statements

Reply:
@DarjaLeonova do you print a string to see the string value of the byte slice?

Reply:
Use `t.Log` to print log messages in the future. When running tests in parallel, `fmt` prints in the middle of all parallel tests running at the time. `t.Log` is attached to the test and is printed in the test output.